content/guidance/become-a-teacher-in-england.md

```diff
@@ -793,67 +545,52 @@ We suggest you use Apply for teacher training if:
 
 ### Preparing to apply for teacher training
 
-The Apply for teacher training service will guide you through the
-application process all the way to enrolment on your course.
+The Apply for teacher training service will guide you through the application process all the way to enrolment on your course.
 
 However there are things you could start to consider right away.
 
 #### Think about why you want to be a teacher
 
-As part of the application process, you’ll be asked to tell training
-providers why you want to be a teacher.
+As part of the application process, you’ll be asked to tell training providers why you want to be a teacher.
 
-This is one of the most important parts of your application, so it’s
-worth thinking about it in advance.
+This is one of the most important parts of your application, so it’s worth thinking about it in advance.
 
-You should use this section to showcase your motivation, commitment and
-teaching potential, backing up your answer with specific examples.
+You should use this section to showcase your motivation, commitment and teaching potential, backing up your answer with specific examples.
 
-Give providers an insight into your personality by writing honestly and
-thoughtfully. Avoid cliché and write in clear, correct, concise English.
+Give providers an insight into your personality by writing honestly and thoughtfully. Avoid cliché and write in clear, correct, concise English.
 
-To get help with personal statements from a teacher training adviser,
-[use the Get an adviser service](/tta-service).
+To get help with personal statements from a teacher training adviser, [use the Get an adviser service](/tta-service).
 
 You don't have to cover everything in this list, but suggested topics
 include:
 
   - why you want to be a teacher
-  - your passion for your subject and the age group you’ve chosen to
-    teach
+  - your passion for your subject and the age group you’ve chosen to teach
   - the welfare and education of children and/or young people
   - the demands and rewards of the profession
   - personal qualities that will make you a good teacher
   - your contribution to the life of the school outside the classroom –
     for example, running extra-curricular activities and clubs
-  - if you have school experience or have worked as a volunteer with
-    children or young people, give details of what this has taught you
+  - if you have school experience or have worked as a volunteer with children or young people, give details of what this has taught you
 
 #### What do you know about the subject you want to teach?
 
-The application form also asks for detailed evidence for the knowledge
-and interest you bring to the subject(s) you’d like to teach.
+The application form also asks for detailed evidence for the knowledge and interest you bring to the subject(s) you’d like to teach.
```

Comment:
Usually following GDS guidance we'd just use the plural here rather than the brackets around the 's'

> Do not use round brackets to refer to something that could either be singular or plural, like ‘Check which document(s) you need to send to DVLA.’

I know this was here prior to your change but it's probably worth tidying up as we go.